packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1058,4 +1060,111 @@ module.exports.external = [
       return PERSONAL_RELATIONSHIP_TYPE_CODE;
     },
   },
+  {
+    name: "decodeHtmlEntities",
+    description: "Decodes html strings",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return "";
+      }
+      const result = he.decode(str);
+      return result;
+    },
+  },
+  {
+    name: "removeNonNumeric",
+    description: "Removes all non-numeric characters from a string, keeping decimal points",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return "";
+      }
+      return str.replace(/[^0-9.]/g, '');
+    },
+  },
+  {
+    name: "convertFeetAndInchesToCm",
+    description: "Checks if a string is in the format 'number ft number in' and if so, converts the feet and inches to centimeters",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return { isValid: false };
+      }
+      const match = str.match(/^(\d+(?:\.\d+)?) ft (\d+(?:\.\d+)?)( in)?$/);
+      if (match) {
+        const feet = parseFloat(match[1]);
+        const inches = parseFloat(match[2]);
+        const cm = (feet * 12 + inches) * 2.54;
+        return { isValid: true, value: cm, unit: 'cm' };
+      } else {
+        return { isValid: false };
+      }
+    },
+  },
+  {
+    name: "extractNumberAndUnit",
+    description: "Checks if a string is in the format 'number unit' and if so, extracts the number and the unit",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return { isValid: false };
+      }
+      const match = str.match(/^(\d+(?:\.\d+)?)(\s*)([a-zA-Z\/]+)$/);
+      if (match) {
+        return { isValid: true, value: parseFloat(match[1]), unit: match[3] };
+      } else {
+        return { isValid: false };
+      }
+    },
+  },
+  {
+    name: "extractComparator",
+    description: "Checks if a string starts with a comparator followed by a decimal number, and if so, extracts the comparator and the number",
```

Comment:
very common for us to get a greater than or less than values in the value. 
"&lt;&#x3D;149" is the same as "<=149"